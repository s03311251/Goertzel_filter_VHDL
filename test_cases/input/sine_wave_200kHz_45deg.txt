36A0
3C82
1AFE
0065
1179
36A0
3C82
1AFE
0065
1179
36A0
3C82
1AFE
0065
1179
36A0
3C82
1AFE
0065
1179
36A0
3C82
1AFE
0065
1179
36A0
3C82
1AFE
0065
1179
36A0
3C82
1AFE
0065
1179
36A0
3C82
1AFE
0065
1179
36A0
3C82
1AFE
0065
1179
36A0
3C82
1AFE
0065
1179
36A0
3C82
1AFE
0065
1179
36A0
3C82
1AFE
0065
1179
36A0
3C82
1AFE
0065
1179
36A0
3C82
1AFE
0065
1179
36A0
3C82
1AFE
0065
1179
36A0
3C82
1AFE
0065
1179
36A0
3C82
1AFE
0065
1179
36A0
3C82
1AFE
0065
1179
36A0
3C82
1AFE
0065
1179
36A0
3C82
1AFE
0065
1179
