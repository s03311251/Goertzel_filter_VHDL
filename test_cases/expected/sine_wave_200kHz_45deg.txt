FFFFFF40
FFFFFF00
